rfc/rfc-61/rfc-61.md

```diff
@@ -0,0 +1,244 @@
+
+
+### Operations
+* Create Snapshot View
+  create savepoint on a specific commit, meanwhile, create a new external table name tablename_YYYYMMDD, add as.of.instant={savepoint's timestamp} into external table storage properties ， this table has the same basepath with the original Hudi table
+```sql
+call create_snapshot_view(table => 'hudi_table', commit_Time => 'commit_timestamp_from_timeline', snapshot_table => 'snapshot_hive_table');
+```
+
+| Parameter Name   | Required | Default                       | Remarks                                                                                       |
+|------------------| -------  |-------------------------------|-----------------------------------------------------------------------------------------------|
+| `table`          | `true`   | `--`                          | the Hive table name you want to create savepoint, must be a Hudi table, without database name |
+| `commit_Time`    | `false`  | `None`                        | the commit timestamp from Hudi timeline, if not provided, will use the latest commit          |
+| `user`           | `false`  | `""`                          | the user name will be saved in Hudi savepoint metadata                                        |
+| `comments`       | `false`  | `""`                          | the comment will be saved in Hudi savepoint metadata                                          |
+| `snapshot_table` | `false`  | `$table name + _$commit_time` | the snapshot view table name in hive                                                          |
+| `hms`            | `false`  | `None`                        | Hive metastore server used for syncing savepoint information                                  |
+
+* Delete Snapshot View
+  call delete savepoint command(via spark-sql or hudi-cli), meanwhile delete associated Hive table
+```sql
+call delete_snapshot_view(table => 'hudi_table', snapshot_table => 'snapshot_hive_table');
+```
+
+|  Parameter Name  | Required | Default | Remarks |
+|  --------------  | -------  | ------- | ------- |
+| `table`          | `true`   | `--`    | the Hive table name you want to delete savepoint, must be a Hudi table |
+| `instant_time`   | `true`   | `--`    | the savepoint timestamp from Hudi timeline, must provide |
+| `hive_sync`      | `false`  | `false` | whether to delete savepoint timestamp from Hive table serde properties |
+| `hms`            | `false`  | `None`  | Hive metastore server used for deleting savepoint information |
+
+* List Snapshot View
+```sql
+call show_snapshotviews(table => 'hudi_table');
+```
+
+|  Parameter Name  | Required | Default | Remarks |
+|  --------------  | -------  | ------- | ------- |
+| `table`          | `true`   | `--`    | the Hive table name you want to show savepoint list, must be a Hudi table |
+
+
+
+## Rollout/Adoption Plan
```

Comment:
One critical fix we might have to do on this regards is supporting archival beyond savepoint. 
https://issues.apache.org/jira/browse/HUDI-3884 
even though the ticket is closed, we have not certified across all diff table states, all diff queries etc. If not, archival will stop at the first savepointed commit. 
w/ 1.x, things might change. CC @danny0405 @codope